src/components/Yritys/YritysForm.svelte

```diff
@@ -28,9 +30,21 @@
 
   export let disabled = false;
 
+  export let luokittelut = Maybe.None();
+
+  $: Future.fork(
+    flashMessageStore.add('Yritys', 'error'),
+    ([laskutuskielet, verkkolaskuoperaattorit]) =>
+      luokittelut = Maybe.Some({laskutuskielet: laskutuskielet,
+                               verkkolaskuoperaattorit: verkkolaskuoperaattorit}),
+    Future.parallel(2,[laskutusApi.laskutuskielet, laskutusApi.verkkolaskuoperaattorit])
+  );
+
```

Comment:
näitä hakuja ei pitäisi tehdä formi tasolla vaan sen sijaan kontrolleritasolla

Reply:
nyt jos haussa tapahtuu virhe niin formin on hankala antaa mitään järkevää virheilmoitusta

Reply:
lisäksi logiikka on tarpeettoman monimutkaista kun luokittelut on maybe-boksissa

Reply:
Nämä on nyt fiksattu